test/FT-voting-quadratic/RIFGovernorFtQuadratic.test.js

```diff
@@ -171,21 +183,64 @@ describe('Governance - Fungible tokens voting', () => {
             reason,
           );
       });
+    });
 
+    describe('Voting results', () => {
       it('total votes should equal voters rif amount square root sum', async () => {
         const againstVotes = await governor.getAgainstVotes(proposalId);
-        expect(againstVotes).to.equal(
-          Math.sqrt(voters[0].rifAmount) + Math.sqrt(voters[1].rifAmount),
-        );
+        expect(againstVotes).to.equal(Math.sqrt(voters[1].rifAmount));
         const forVotes = await governor.getForVotes(proposalId);
-        expect(forVotes).to.equal(Math.sqrt(voters[2].rifAmount));
-        console.log(forVotes, againstVotes);
+        expect(forVotes).to.equal(
+          Math.sqrt(voters[2].rifAmount) + Math.sqrt(voters[0].rifAmount),
+        );
+      });
```

Comment:
With the assertions here, I suggest that you use "easy" numbers. For example 1, 100, and 10000, ... or 2, 4, 16 ... etc, so that the votes are easy to check manually, and it is possible to write assertions with literal values specified for the expected values.